### PR TITLE
Image Classification with AlexNet - Example

### PR DESCRIPTION
**Pull request by sayannath**
Hello @fchollet added an example of AlexNet Image Classification on CIFAR-10 dataset.

Here is the Colab notebook - [Link](https://colab.research.google.com/drive/1DSKT2Y8hnZ4uYjVMhMZWwIL77_cMvo0b?usp=sharing)

### CODE REVIEW

File: examples/vision/alexnet.py [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+    model = keras.Sequential(
+        [
+            keras.layers.Conv2D(
+                filters=96,
+                kernel_size=(11, 11),
+                strides=(4, 4),
+                activation="relu",
+                input_shape=(IMAGE_SIZE, IMAGE_SIZE, 3),
+            ),
+            keras.layers.BatchNormalization(),
```
Comment (fchollet) [inline examples/vision/alexnet.py:152]: AlexNet did not include BatchNormalization.

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

Overall, AlexNet has historical interest, but is very far from modern best practices. Ideally, we'd like our examples to illustrate modern best practices: users should be able to modify / copy them to solve their own problems.

Further, this example doesn't really add much over [something like this](https://keras.io/examples/vision/mnist_convnet/) (which is from 2015).

I would not recommend merging this example. Instead, I'd point you to this list of topics that would be great to add to our list: https://github.com/keras-team/keras-io/blob/master/call_for_contributions.md

**Comment by sayannath** [comment]
Thanks for the advice. Closing this PR.
